Init suicides in latest docker build

#### Expected behavior
Successful bootup

#### Actual behavior
Kernel panic

#### Steps to reproduce
- Run the Sep 28, 2018 build (current latest): `docker run harveyos/harvey@sha256:d4371ca1082cf9f7b19a510a4ea487b9038062d5d03ded9d2cb5a71e94b26be` 

#### QEMU output
```
qemu-system-x86_64 -s -cpu Opteron_G1 -smp 4 -m 2048 -nographic --machine q35 -net nic,model=rtl8139 -net user,id=user.0,hostfwd=tcp::5555-:1522,hostfwd=tcp::9999-:9,hostfwd=tcp::17010-:17010,hostfwd=tcp::1666-:1666,hostfwd=tcp::5356-:5356,hostfwd=tcp::17013-:17013 -object filter-dump,id=f1,netdev=user.0,file=/tmp/vm0.pcap -append "service=cpu nobootprompt=tcp maxcores=1024 fs=10.0.2.2 auth=10.0.2.2 nvram=/boot/nvram nvrlen=512 nvroff=0 acpiirq=1" -kernel /harvey/sys/src/9/amd64/harvey.32bit

Harvey
mmuinit: vmstart 0xfffffffff0000000 vmunused 0xfffffffff095e000 vmunmapped 0xfffffffff0c00000 vmend 0xfffffffff4000000
sys->pd 0x108003 0x108023
cpu0: mmu l3 pte 0xfffffffff0106ff8 = 107023
cpu0: mmu l2 pte 0xfffffffff0107ff8 = 108023
cpu0: mmu l1 pte 0xfffffffff0108c00 = e3
cpu0: mmu l1 pte 0xfffffffff0108c00 = e3
base 0xfffffffff0966000 ptr 0xfffffffff0966000 nunits 3578369
allocated
 free ok
parsexsdt: tbl fffffffff096804c, end 16
ACPI initialized
	acpiinit();
physalloc color=0 base=0x4000000 size=0x7bfdf000
before mpacpi, maxcores 1024
APIC lapic paddr 0xfee00000, flags 0x00000001
ASlapic 0
apic proc 0/0 apicid 0 
ASlapic 1
apic proc 1/1 apicid 1 
ASlapic 2
apic proc 2/2 apicid 2 
ASlapic 3
apic proc 3/3 apicid 3 
ASioapic 0
ioapic 0 addr fec00000 ibase 0 
Could not find an IOAPIC for global irq 62!
Unable to build IOAPIC route for irq 62
intrenable: couldn't enable irq 62, tbdf 0x5ffff00 for #IPI
apic0: hz 1000568060 max 10005680 min 100056 div 3
sipiptr 0xfffffffff0003000 sipipa 0x3000
p 0xfffffffff097e000 sipiptr[-1] 0x97e000
p 0xfffffffff098b000 sipiptr[-1] 0x98b000
p 0xfffffffff0998000 sipiptr[-1] 0x998000
ioapicintrinit: success
virtio: initializing
virtio: no devices
bus dev type vid  did intl memory
0   0/0 06 00 00 8086 29c0 x   0  
0   1/0 03 00 00 1234 1111 x   0  0:fd000008 16777216 2:febd0000 4096 
0   2/0 02 00 00 10ec 8139 A  11  0:0000c001 256 1:febd1000 256 
0  31/0 06 01 00 8086 2918 x   0  
0  31/2 01 06 01 8086 2922 A  10  4:0000c141 32 5:febd2000 4096 
0  31/3 0c 05 00 8086 2930 A  10  4:00000701 64 
ioapicintrinit: success
ioapicintrinit: success
#l0: rtl8139: 100Mbps port 0xc000 irq 11 tu 1514: 525400123456
#S/sdE: ich: 0xfebd2000 sata 1, 6 ports, irq 10
unknown color for cpu0
cpu1 color -1 role TCcpu2 color -1 role TC tsc 7318155500
schedinit: no sch for cpu2
unknown color for cpu2
Scan fffffffff0000000
cpu3 color -1 role TC tsc 7333557312
schedinit: no sch for cpu3
unknown color for cpu3
 tsc 7316445593
schedinit: no sch for cpu1
unknown color for cpu1
bootcore: all corScan fffffffff00f0000
get_coreboot_info: ret -1
es done
CPU Freq. 2795MHz
schedinit...
virtio-9p initializing
virtio-serial-pci initializing
*init* 1: suicide: invalid address 0xfffffffff14aa1a0/8 in sys call pc=0x20016f
panic: cpu2: boot process died: Suicide
die:wait forever
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM alpine:3.5
+FROM alpine:latest
 
 RUN apk update; apk add git git-perl go gcc bison musl-dev qemu qemu-system-x86_64 curl bash
 ENV HARVEY=/harvey ARCH=amd64 CC=gcc

